how-to/README.md

```diff
@@ -1,11 +1,22 @@
-# Howto guides
+# HowTo Guides
```

Comment:
The spell checker won't like this I'm afraid. If you think we need the camel-case version, please raise a PR to add it to:

- https://github.com/kata-containers/tests/blob/master/cmd/check-spelling/data/main.txt

Reply:
Let's stick with `Howto`  for now :laughing: